### DISCUSSION TITLE
nemo - add selected items to new folder

### DISCUSSION THREAD

**Discussion by decuser** [linuxmint/discussions#60] [Ideas]
I think it'd be swell if when the user has file(s) selected, that the right click menu would have an option to add selected files to a new unnamed folder, that they then would be able to rename. This is how finder works, and it's a great feature.

**Comment by Secret-chest** [comment]
It would be nice to have this built-in as an action.

**Comment by axel358** [comment]
Nautilus has the option as well and it's very useful
